### ISSUE TITLE
MultiInput: Accessibility: Unable to access tokens with keyboard

### ISSUE THREAD

**Issue by agauyeung** [SAP/ui5-webcomponents-react#1307]
**Describe the bug**
MultiInput tokens are not keyboard accessible.

**Isolated Example**
https://codesandbox.io/s/flamboyant-shannon-2lo6j

**To Reproduce**
1. Go to https://codesandbox.io/s/flamboyant-shannon-2lo6j
2. Click into the multi-input box
3. Press the left arrow key to navigate to a token
4. Issue: notice that the focus does not move to the token

**Expected behavior**
Tokens should be focusable.

**UI5 Web Components for React Information**
"@ui5/webcomponents": "0.29.4"
"@ui5/webcomponents-fiori": "0.29.0"
"@ui5/webcomponents-react": "0.129.3"
Operating System: Windows
Browser: Chrome

**Additional context**
This is not reproducible in other versions:
"@ui5/webcomponents-react": "0.13.4"

**Comment by MarcusNotheis** [comment]
Hey @agauyeung,

good catch, thanks! Fixed and released with `@ui5/webcomponents-react@0.129.4`
